assets/src/scss/pages/post/_comments-block.scss

```diff
@@ -55,6 +55,7 @@
     .comment-reply-link {
       float: right;
       margin-top: 0;
+      border-bottom: none;
```

Comment:
I don't think this is needed, since we add the border bottom only when the new identity styles are enabled? And then you can probably remove the `!important` from [here](https://github.com/greenpeace/planet4-master-theme/pull/1959/files#diff-b418b2241107b99a37ed0291b0385d4ad8fe3762dd4d859be83bf83c070428d7R129)

Reply:
Good catch! You're always correct 🥇